openconcept/propulsion/systems/thermal_series_hybrid.py

```diff
@@ -1,56 +1,51 @@
 
 from __future__ import division
```

Comment:
We might want to look and see how this is being used in OC, if I remember correctly this is a relic of python 2.